docker/app
# [WIP][TAR-995 ]ADDED License scan stage to occur pre-commit in Jenkins

Signed-off-by: Zuhayr Elahi <elahi.zuhayr@gmail.com>



**- What I did**
I added a stage which executes License scans against `docker/app`

**- How I did it**
There were two Makefile commands that were added: `fossa-analyze` and `fossa-test`.  The analyze executes a command which uploads a fossa report here: 
https://app.fossa.com/projects/custom%2B11%2Fgit%40github.com%3Adocker%2Fapp/refs/branch/master/ceac1cf22d174a9ed8118f03b122024c7459d3b4

After executing a scan, it then performs the `fossa test` command which looks at the scan for license violations

**- How to verify it**

I executed the `fossa-analyze` command locally: 
`BRANCH_NAME=master make fossa-analyze`

Then executed `fossa-test` which:
`make fossa-test`

The other way I verified that this worked was through running the CI.  It is currently a Work in progress, but once we verify it works on the ci platform it should be ready to merge

**- Description for the changelog**
Execute scan for license issues on project commit


**- A picture of a cute animal (not mandatory but encouraged)**
![image](https://user-images.githubusercontent.com/5545603/63615280-d2dedd80-c599-11e9-96bf-cf95e72cff83.png)
